package.json

```diff
@@ -9,118 +9,85 @@
+  ],
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "oclif": {
+    "commands": "./lib/commands",
+    "bin": "mythxjs",
+    "plugins": [
+      "@oclif/plugin-help"
     ]
   },
-  "config": {
-    "commitizen": {
-      "path": "node_modules/cz-conventional-changelog"
-    }
-  },
-  "jest": {
-    "transform": {
-      ".(ts|tsx)": "ts-jest"
-    },
-    "testEnvironment": "node",
-    "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|tsx|js)$",
-    "moduleFileExtensions": [
-      "ts",
-      "tsx",
-      "js"
-    ],
-    "coveragePathIgnorePatterns": [
-      "/node_modules/",
-      "/test/"
-    ],
-    "coverageThreshold": {
-      "global": {
-        "branches": 90,
-        "functions": 95,
-        "lines": 95,
-        "statements": 95
-      }
-    },
-    "collectCoverageFrom": [
-      "src/*.{js,ts}"
-    ]
+  "scripts": {
+    "clean": "rimraf .temp node_modules/mocha/.temp",
+    "lint": "tslint -t codeFrame 'src/**/*.ts' 'test/**/*.ts'",
+    "lint:fix": "tslint --fix -t codeFrame 'src/**/*.ts' 'test/**/*.ts'",
+    "postpack": "rm -f oclif.manifest.json",
+    "posttest": "tslint -p test -t stylish",
+    "prepack": "rm -rf lib && tsc -b && oclif-dev manifest && oclif-dev readme",
+    "prettier-ts": "prettier --write --parser typescript \"{src,test}/**/*.ts\"",
+    "test": "yarn clean && nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\"",
+    "version": "oclif-dev readme && git add README.md"
   },
-  "prettier": {
-    "semi": false,
-    "singleQuote": true
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
   },
-  "commitlint": {
-    "extends": [
-      "@commitlint/config-conventional"
+  "lint-staged": {
+    "{src,test}/**/*.ts": [
+      "prettier --write --parser typescript",
+      "yarn lint:fix",
+      "git add"
     ]
-  },
-  "devDependencies": {
-    "@commitlint/cli": "^7.5.2",
-    "@commitlint/config-conventional": "^7.5.0",
-    "@types/jest": "^24.0.11",
-    "@types/node": "^11.11.6",
-    "colors": "^1.3.3",
-    "commitizen": "^3.0.7",
-    "coveralls": "^3.0.3",
-    "cross-env": "^5.2.0",
-    "cz-conventional-changelog": "^2.1.0",
-    "husky": "^1.3.1",
-    "jest": "^24.5.0",
-    "jest-config": "^24.5.0",
-    "lint-staged": "^8.1.5",
-    "lodash.camelcase": "^4.3.0",
-    "prettier": "^1.16.4",
-    "prompt": "^1.0.0",
-    "replace-in-file": "^3.4.4",
-    "rimraf": "^2.6.3",
-    "rollup": "^1.7.3",
-    "rollup-plugin-commonjs": "^9.2.2",
-    "rollup-plugin-json": "^4.0.0",
-    "rollup-plugin-node-resolve": "^4.0.1",
-    "rollup-plugin-sourcemaps": "^0.4.2",
-    "rollup-plugin-typescript2": "^0.20.1",
-    "semantic-release": "^15.13.3",
-    "shelljs": "^0.8.3",
-    "travis-deploy-once": "^5.0.11",
-    "ts-jest": "^24.0.0",
-    "ts-node": "^8.0.3",
-    "tslint": "^5.14.0",
-    "tslint-config-prettier": "^1.18.0",
-    "tslint-config-standard": "^8.0.1",
-    "typedoc": "^0.14.2",
-    "typescript": "^3.3.4000"
   }
```

Comment:
I think this whole file can be cleaned up and we get rid of some unused packages and other commands that are not relevant

Reply:
Everything is needed by the oclif framework and have been installed by them, by default. Rest dependencies which have been added are being used.